tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -16073,6 +16141,51 @@ TEST_F(EnergyPlusFixture, Test_UnitarySystemModel_SubcoolReheatCoil)
     coilOutletEnthalpy = Psychrometrics::PsyHFnTdbW(coilOutletTemp, coilOutletHumRat);
     coilOutletTSat = Psychrometrics::PsyTsatFnHPb(*state, coilOutletEnthalpy, state->dataEnvrn->OutBaroPress);
     EXPECT_TRUE(coilOutletTemp >= coilOutletTSat);
+
```

Comment:
New additions to unit test, same results as before up to this point. Check that the ComponentIndex and compPointer are looking at the correct object for the 2 new OA system objects added to this unit test OA equipment list (pointer based UnitarySystem objects).